### PR TITLE
Review page now writes answers to the database for reviewed user

### PR DESCRIPTION
**Pull request by gw5045**
The review page can now write the answers input to the database of the user being reviewed.

### CODE REVIEW

**Comment by gw5045** [comment]
Only one review can be stored at a time. If we were to continue working on this, i would make it so reviews from multiple users would store to display.

File: SoulMatchServer-main/src/main/java/com/soulmatch/model/Profile.java [hunk 1]
```diff
@@ -37,6 +48,16 @@ public Profile(String gender, String bio, String picture, List<String> profilePi
 
     public int getScore() { return score; }
 
+    public String getReviewAns1() { return reviewAns1; }
+
+    public String getReviewAns2() { return reviewAns2; }
+
+    public String getReviewAns3() { return reviewAns3; }
```
Comment (mahinhaque27) [inline SoulMatchServer-main/src/main/java/com/soulmatch/model/Profile.java:55]: Variable names should be more specific

File: SoulMatchServer-main/src/main/java/com/soulmatch/model/reviewQuestions.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
+package com.soulmatch.model;
+
```
Comment (mahinhaque27) [inline SoulMatchServer-main/src/main/java/com/soulmatch/model/reviewQuestions.java:2]: Array should be used for future scalability of questions

**Review by mahinhaque27 — APPROVED** [review]
LGTM

**Pull request merged**